src/organizations/components/OrgProfileTab/index.tsx

```diff
@@ -73,6 +78,8 @@ const OrgProfileTab: FC = () => {
     retrieveOrgDetails()
   }, [dispatch, orgDetailsStatus, quartzOrg.id])
 
+  const allowSelfRemoval = users.length > 1
+  const showLeaveOrgButton = isFlagEnabled('createDeleteOrgs')
```

Comment:
I spent some time thinking about whether this check - `allowSelfRemoval` - is sufficient for multi-org accounts. I think it is.

We have additional checks like `isSuspendable` to prevent whether a user can _delete_ the last org in the account. That's necessary because we don't want an orphaned account without any orgs in it.

I don't think we have that problem here as long as we check if `users.length > 1` - if a user tries to remove themselves from an org, even if it's the last org, that check will require the presence of another user. So, barring unusual circumstances like two users removing themselves simultaneously, it shouldn't end up being possible for all users to get removed from the last org.